### ISSUE TITLE
Why new version changelog still include previous version release changlog?

### ISSUE THREAD

**Issue by lovetingyuan** [conventional-changelog/standard-version#893]
` npx standard-version -p beta --preMajor -r patch  --skip.tag --header="### Wyxc" --path ./ `

### Features

* **huhu:** sfsd 9c5c0cd

### Bug Fixes

* **sdfs:** dsjfkds ef048c3

### Features

* **huhu:** sfsd 9c5c0cd

### Features

* **huhu:** sfsd 9c5c0cd

**Comment by thetaPC** [comment]
It seems that an answer was given in [issue 628.](https://github.com/conventional-changelog/standard-version/issues/628)

**Comment by JJetmar** [comment]
> It seems that an answer was given in [issue 628.](https://github.com/conventional-changelog/standard-version/issues/628)

Yes, this was exactly my case. It seems like standard-version use git tags to distinguish changes between versions for generating a changelog.

In my case, we used just the version number (without the "v" prefix which is default for standard-version) for a git tag. So the solution was to set an empty string for the tag via option `--tag-prefix ""` when running standard-version.

But I am not sure about standard-version works correctly if you are not using git tags at all :\
